.github/workflows/ci_tests.yml

```diff
@@ -345,6 +348,14 @@ jobs:
           exit 1
         fi
 
+    - name: Validate inputs
+      run: |
+        if [[ "${framework}" == "mkdocs" && ! "${{ inputs.python_version_docs}}" =~ ^3\.([7-9]|1[0-3])(\..*)?$ ]]; then
```

Comment:
Hmm. Good question.
I set this up only where we're calling _this_ package, i.e., `ci-cd`. And in the steps "Update API Reference" and "Update landing page" further down, the `ci-cd` package is used, but _only_ for the `mkdocs` framework.

In theory, one could use the `sphinx` framework together with Python 3.6 (not supported by `ci-cd`) and still have the workflow run fine (so long as the Python dependencies they use support Python 3.6).

Hence, I've set this up to only check for the `mkdocs` framework, because I know that the Python version limitation here is _at least_ that of the `ci-cd` package. It may be even more strict (if the user requests dependencies that for example do not support Python 3.7), but it may _not_ be more loose, as then the `ci-cd` package cannot run.